src/components/CreateResetPassword/CreateResetPassword.js

```diff
@@ -13,6 +13,10 @@ import isEmpty from 'lodash/isEmpty';
 import remove from 'lodash/remove';
 import some from 'lodash/some';
```

Comment:
I suggest to combine these imports into single one
`import {
  remove, 
  some, 
  isEmpty
} from lodash`